plancton/__init__.py

```diff
@@ -210,18 +211,19 @@ def _read_conf(self):
       self.conf["max_docks"] = 0
     if not isinstance(self.conf["docker_cmd"], list):
       self.conf["docker_cmd"] = self.conf["docker_cmd"].split(" ")
-    if self.conf["influxdb_url"] and not "#" in self.conf["influxdb_url"]:
-      # Invalid InfluxDB URL: disable monitoring
-      self.conf["influxdb_url"] = None
+    tmp_url_list = []
+    if isinstance(self.conf["influxdb_url"], str):
+      self.conf["influxdb_url"] = [ self.conf["influxdb_url"] ]
+    if isinstance(self.conf["influxdb_url"], list):
+      tmp_url_list = [x for x in self.conf["influxdb_url"] and "#" in x]
+    self.conf["influxdb_url"] = tmp_url_list
     self.logctl.debug("Configuration:\n%s" % json.dumps(self.conf, indent=2))
 
   # Set up monitoring target.
   def _influxdb_setup(self):
-    if not self.conf["influxdb_url"]:
-      self.streamer = lambda *x,**y: True
-      return
-    baseurl,db = self.conf["influxdb_url"].split("#")
-    self.streamer = InfluxDBStreamer(baseurl=baseurl, database=db)
+    self.streamers = [InfluxDBStreamer(baseurl=url.split("#")[0], database=url.split("#")[1]) \
```

Comment:
Those lines are utterly unreadable, this is Perl in Python disguise. Also, there are ways of using _all_ the values returned by `split()` instead of calling it multiple times for having only some elements out (see Python's builtin `zip()` for instance).

Here you need to init an `InfluxDBStreamer` for every URL you have. Why don't you just:

```python
self.streamers = []
for url in self.conf["influxdb_url"]:
  self.streamers.append(InfluxDBStreamer(**dict(zip(["baseurl", "database"], url.split("#", 1)))))
```